testing/tests/DevExpress.ui.widgets.editors/datebox.tests.js

```diff
@@ -4777,6 +4777,23 @@ QUnit.module('keyboard navigation', {
         assert.ok($cancelButton.hasClass('dx-state-focused'), 'cancel button is focused');
     });
 
+    QUnit.testInActiveWindow('Popup has open state if tab key was pressed when applyValueMode: "instantly"', function(assert) {
```

Comment:
Usually `testInActiveWindow` method uses for testing of `focus`. Is it necessary to add it?

Reply:
Yes, exactly! Fixed in [99a178d2a43ec5be8ad8dd319a440caaa9118077].